src/api/examples.md

```diff
@@ -1484,48 +1484,50 @@ AssignMeshToTeam(cube, 1)
 
 ### <a id="event:CoreObject.childAddedEvent"></a>CoreObject.childAddedEvent
 
+This event fires when something gets added inside of an object directly, not inside of any of it's children.
+
+```lua
+local obj = script.parent -- The object that will be affected by this event.
+
+obj.childAddedEvent:Connect(function() -- This will run when a child gets added to the object.
+    -- Code to run
+end)
+```
+
 ### <a id="event:CoreObject.childRemovedEvent"></a>CoreObject.childRemovedEvent
 
-### <a id="event:CoreObject.descendantAddedEvent"></a>CoreObject.descendantAddedEvent
+This event fires when something gets removed from an object directly, not inside of any of it's children.
 
-### <a id="event:CoreObject.descendantRemovedEvent"></a>CoreObject.descendantRemovedEvent
+```lua
+local obj = script.parent -- The object that will be affected by this event.
```

Comment:
We've been trying to avoid using `script.parent` as a core object in these samples, for a couple of reasons.

First, because we don't know what the hierarchy looks like, and we want to avoid potential side effects if people just plunk the examples into the game.  (Admittedly side effects from attaching an event listener are minor compared to, say, an example that called `script.parent:Destroy()` or something.  But we like them to be consistent.)

Second though, these scripts also get run by an automated process that verifies that all the examples work before publishing.  And due to how it is set up, `script.parent` is not guaranteed to be a good thing to use in samples.

Anyway!  TL;DR:
If a sample needs a `CoreObject` to operate on, we'd prefer if the code creates it itself (via something like `local template1 = World.SpawnAsset(propCubeTemplate)` than use `script.parent`.